src/hardware/link-mso19/protocol.c

```diff
@@ -24,38 +25,34 @@
 
 /* serial protocol */
 #define mso_trans(a, v) \
-	(((v) & 0x3f) | (((v) & 0xc0) << 6) | (((a) & 0xf) << 8) | \
+	g_htons(((v) & 0x3f) | (((v) & 0xc0) << 6) | (((a) & 0xf) << 8) | \
```

Comment:
I believe there is a clearer alternative than g_htons for managing endianness. Ran out of time to look it up, sorry

Reply:
that would be `read_be16` and similar funcs